docs/mutations.rst

```diff
@@ -0,0 +1,219 @@
+Mutations
+=========
+
+Introduction
+------------
+
+Graphene-Django makes it easy to perform mutations.
+
+With Graphene-Django we can take advantage of pre-existing Django features to
+quickly build CRUD functionality, while still using the core `graphene mutation <https://docs.graphene-python.org/en/latest/types/mutations/>`__
+features to add custom mutations to a Django project.
+
+Simple example
+--------------
+
+.. code:: python
+
+    import graphene
+
+    from graphene_django import DjangoObjectType
+
+    from .models import Question
+
+
+    class QuestionType(DjangoObjectType):
+        class Meta:
+            model = Question
+
+
+    class QuestionMutation(graphene.Mutation):
+        class Input:
+            # The input arguments for this mutation
+            text = graphene.String(required=True)
+            id = graphene.ID()
+
+        # The class attributes define the response of the mutation
+        question = graphene.Field(QuestionType)
```

Comment:
Can also do `Output = QuestionType` and then just return the question instance in the `mutate` method

Reply:
Well today I learned! See, this is why we need the better docs 😆 I'll add this